alacritty.yml

```
@@ -375,7 +375,7 @@
     # This program is executed when clicking on a text which is recognized as a URL.
     # The URL is always added to the command as the last parameter.
     #
-    # When set to `None`, URL launching will be disabled completely.
+    # When launcher is set to None, URL launching will be disabled completely.
```

I think it might be a bit better to state the option even more explicitly, to make sure **everyone** gets it:

```suggestion
    # When set to `launcher: None`, URL launching will be disabled completely.
```

---

Great idea!